tsconfig.cypress.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "types": ["cypress", "@testing-library/cypress", "./cypress/support"],
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": "./src",
```

Comment:
I guess this is working, but is this really valid JSON (the trailing comma)?

Reply:
I fixed it :)